### PR TITLE
Fix error code for every error

### PR DESCRIPTION
**Pull request by ritters**
Closes GH-10

@mmoll @xalopp @andythedandy @calbrecht please review

### CODE REVIEW

File: Sniffs/Commenting/PropertyCommentSniff.php [hunk 1]
```diff
@@ -94,7 +94,9 @@ protected function processTokenWithinScope(
             array(T_DOC_COMMENT_OPEN_TAG),
             $stackPtr
         );
-        if ($postComment !== false && $tokens[$postComment]['line'] === $tokens[$stackPtr]['line']) {
+        if ($postComment !== false
+            && $tokens[$postComment]['line'] === $tokens[$stackPtr]['line']
```
Comment (mmoll) [inline Sniffs/Commenting/PropertyCommentSniff.php]: I would like to see the ```&&``` in the line above, so I know, the statement will continue in the next line.

Comment (ritters) [inline Sniffs/Commenting/PropertyCommentSniff.php]: PHPCS code style wants the boolean operator on the new line:
Each line in a multi-line IF statement must begin with a boolean operator

Comment (mmoll) [inline Sniffs/Commenting/PropertyCommentSniff.php]: I see. it's an outrage :boom:

**Comment by xalopp** [comment]
looks good

**Comment by mmoll** [comment]
thanks!

**Pull request merged**